GTViewController/Classes/Custom Classes/GTInputFieldView.m

```diff
@@ -14,13 +14,19 @@
 #import "GTInputFieldView.h"
 #import "GTLabel.h"
 
+@interface GTInputFieldView ()
+
+@property (strong, nonatomic) NSString              *inputFieldType;
+@property (strong, nonatomic) UITextField           *inputTextField;
+@end
+
 @implementation GTInputFieldView
 
 
 - (instancetype)createInputFieldFromElement:(TBXMLElement *)element withY:(CGFloat)yPos withStyle:(GTPageStyle*)style presentingView:(UIView *)presentingView {
     
     GTLabel *inputFieldLabel = nil;
-    UITextField *inputTextField = [[UITextField alloc]init];
+    self.inputTextField = [[UITextField alloc]init];
```

Comment:
This is a small thing but could you add the space in front of the init method `[[UITextField alloc] init];` it improves readability.